README.md

```diff
@@ -0,0 +1,29 @@
+# How are the docs on openliberty.io built and what is a docs playbook?
+
+The docs on https://openliberty.io/ are built using a tool called Antora(https://antora.org/). It is a tool that helps streamline writing, managing, and versioning documentation.
+
+The docs playbook is a key file used by Antora see https://docs.antora.org/antora/latest/playbook/. It tells the build what branches and versions of the [docs repo](https://github.com/OpenLiberty/docs) to use. Antora aggregates all of the content specified in the playbook and creates versions of pages based on the `version` attribute in the `antora.yml` of each doc branch. See https://github.com/OpenLiberty/docs/blob/vNext/antora.yml for an example. If Antora finds the same doc in multiple doc branches, it will create a version picker on the site that lets you easily switch between versions of the doc.
+
+There are three doc playbooks used in the UI builds: prod, staging, and draft.
+
+https://openliberty.io/ is built using the [prod playbook](https://github.com/OpenLiberty/docs-playbook/blob/prod/antora-playbook.yml). This file explicitly only contains versions of the documentation for released Open Liberty versions.
+
+The [staging site](http://staging-openlibertyio.mybluemix.net/) is built using the [staging playbook](https://github.com/OpenLiberty/docs-playbook/blob/staging/antora-playbook.yml). This playbook contains the `staging` branch which is used to stage in changes for the next documentation version to be released. It also contains a wildcard `V*.0.0.*-staging` rule which pulls in all of the branches used to stage in changes to already released versions of documentation.
+
+The [draft site](https://draft-openlibertyio.mybluemix.net/) is built using the [draft playbook](https://github.com/OpenLiberty/docs-playbook/blob/draft/antora-playbook.yml). This playbook contains the `draft` branch which pulls in all of the content currently being drafted. It also contains a wildcard `V*.0.0.*-draft` which pulls in all of the branches used to draft changes to already released versions of documentation.
+
+# Publishing a new release of Open Liberty Docs
+
+This is the flow that must be followed when releasing a new version of docs on openliberty.io. These steps also update the `vNext`, `staging` and `draft` branches so that the next version of docs can be worked on and ensures that fixes can still be made to the docs of the version being released.
```

Comment:
These steps also update the `vNext`, `staging` and `draft` branches of the (docs repo) [https://github.com/OpenLiberty/docs] so that the......